### PR TITLE
Add lighten variables

### PR DESCRIPTION
**Pull request by holic**
See https://github.com/holic/basscss-color-lighten and discussion at https://github.com/holic/basscss-color-lighten/issues/1

### CODE REVIEW

**Comment by jxnblk** [comment]
Looks great, thanks!

File: index.css [hunk 1]
```diff
@@ -42,6 +42,11 @@
   --darken-3: rgba(0,0,0,.25);
   --darken-4: rgba(0,0,0,.5);
 
+  --lighten-1: rgba(0,0,0,.0625);
```
Comment (jxnblk) [inline index.css]: Just realized that these should be `rgba(255,255,255,.0625)`

Comment (holic) [inline index.css]: Oops. Copy pasta error when working too fast.

Comment (holic) [inline index.css]: How would you like me to handle that? Open another PR?

Comment (jxnblk) [inline index.css]: Don't worry I patched it already :)

Comment (holic) [inline index.css]: Cheers!

**Pull request merged**